### ISSUE TITLE
Does not work over code-server

### ISSUE THREAD

**Issue by lijiaqigreat** [coq-community/vscoq#257]
Hi,
   It seems that vscoq does not work over remote vscode using code-server. While doing the same thing on local vscode works.
Repro step:
Install coq using apt (version 8.11)
Install code-server using official install script (vscode version: 1.30)
Run code-server
Install vscoq in official store
Restart code-server
open a simple .v file
Run "Open proof view"
**vscode does nothing.**
Interpret some simple code.
**vscoq shows it's interpreting and hangs forever.**

**Comment by fakusb** [comment]
Very basic question, as I have not used code-server: code-server and coq are installed and running on the same physical machine?

**Comment by lijiaqigreat** [comment]
Correct. The host installs code-server, coq, and almost everything. The client uses vscode in browser without installing anything to the client OS.

**Comment by fakusb** [comment]
Ok, as far as I see, this would require some rewriting, as for example we currently access the file system of the host in the extension using the node-fs-package. (for example, to look for the _CoqProject).

**Comment by Blaisorblade** [comment]
I've learned a (relevant?) alternative from @gstew5: the "Dev Containers" extension can deal with VsCoq by installing it _inside_ the container:
 https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers

(I'm on M1 and run into https://github.com/microsoft/vscode-remote-release/issues/8169, but that issue lists workarounds).
